src/styles/blocks/icons.scss

```diff
@@ -0,0 +1,33 @@
+
+.icon {
+  display: flex;
+  vertical-align: middle;
+  height: 24px;
+  width: 24px;
+  background-image: url(../images/icons/icon-burger-menu.svg);
```

Comment:
Remove the 'background-image' property from the .icon class. It is redundant since it is already defined in the --menu modifier.